Tests/UnitTests/Purchasing/Purchases/BasePurchasesTests.swift

```diff
@@ -101,54 +101,20 @@ class BasePurchasesTests: TestCase {
         // this level it should be moved to `StoreKitUnitTests`, which runs serially.
         Purchases.logLevel = .verbose
 
-        // See `addTeardownBlock` docs:
-        // - These run *before* `tearDown`.
-        // - They run in LIFO order.
         self.addTeardownBlock {
-            expect { [weak purchases = self.purchases] in purchases }
-                .toEventually(beNil(), description: "Purchases has leaked")
-        }
-        self.addTeardownBlock {
-            expect { [weak orchestrator = self.purchasesOrchestrator] in orchestrator }
-                .toEventually(beNil(), description: "PurchasesOrchestrator has leaked")
-        }
-        self.addTeardownBlock {
-            expect { [weak deviceCache = self.deviceCache] in deviceCache }
-                .toEventually(beNil(), description: "DeviceCache has leaked: \(self)")
-        }
+            weak var purchases = self.purchases
+            weak var orchestrator = self.purchasesOrchestrator
+            weak var deviceCache = self.deviceCache
```

Comment:
Should we rename these 3 so there is less confusion with the global variables?

Reply:
They're limited to the scope of this method so I think it's clear that they're shadowing those variables. But let me know if you disagree.